src/software/thunderscope/replay/proto_player.py

```diff
@@ -419,9 +509,19 @@ def seek(self, seek_time: float) -> None:
         # with a timestamp less than (but closest to) the seek_time we want
         # to seek to.
         def __bisect_chunks_by_timestamp(chunk: str) -> None:
```

Comment:
The return type should be updated. Also the name of this function and the associated comment is kinda confusing -- maybe `__get_timestamp_for_chunk` would be better

Reply:
Done